examples/lit/src/composite-form/composite-form.ts

```diff
@@ -0,0 +1,108 @@
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { Button } from '@sl-design-system/button';
+import { ButtonBar } from '@sl-design-system/button-bar';
+import { Checkbox, CheckboxGroup } from '@sl-design-system/checkbox';
+import { Form, FormController, FormField, FormValidationErrors } from '@sl-design-system/form';
+import { Radio, RadioGroup } from '@sl-design-system/radio-group';
+import { TextField } from '@sl-design-system/text-field';
+import { Textarea } from '@sl-design-system/textarea';
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+import styles from './composite-form.scss.js';
+
+export class CompositeForm extends ScopedElementsMixin(LitElement) {
+  /** @internal */
+  static get scopedElements(): ScopedElementsMap {
+    return {
+      'sl-button': Button,
+      'sl-button-bar': ButtonBar,
+      'sl-checkbox': Checkbox,
+      'sl-checkbox-group': CheckboxGroup,
+      'sl-form': Form,
+      'sl-form-field': FormField,
+      'sl-form-validation-errors': FormValidationErrors,
+      'sl-radio': Radio,
+      'sl-radio-group': RadioGroup,
+      'sl-text-field': TextField,
+      'sl-textarea': Textarea
```

Comment:
Are we going to change component name from `sl-textarea` to `sl-text-area`?

Reply:
I mean in general :) not only here

Reply:
Yes, want to do that after this :)